docs/data/joy/components/checkbox/ExampleChoiceChipCheckbox.js

```diff
@@ -49,13 +49,15 @@ export default function ExampleChoiceChipCheckbox() {
                       setValue((val) => val.filter((text) => text !== item));
                     }
                   }}
-                  sx={{
```

Comment:
While improving the support for `componentsProps` in all the Joy UI components sounds great. I wonder if using it for customization over the CSS selectors is a step forward. Personally, I found the previous approach better, as true customization should likely happen with the `styled()` API, which is now harder to migrate to.

Reply:
I think this is not up to Joy and I don't even think which one is better. It's all up to the developers to choose the way they prefer. I will ensure that both ways are documented.

Reply:
> It's all up to the developers to choose the way they prefer.

It's an interesting perspective. I had tried this path in the past, and stopped once I saw too many developers struggling with deciding which option they should use. I saw them ask for the documentation to give them "the way". For example, in our last developer survey: https://www.notion.so/mui-org/Raw-data-aa374141dcb3453dbfea301dcc437126#8830e74c200946dcb29ce385684f1dc4

   10 | more opinionated
    2 | less opinionated
 
Maybe it would make sense to first figure out which approach is better, and then stick to it 😁